test/code-fixtures/apex/YetAnotherTestClass.cls

```diff
@@ -1,20 +1,20 @@
 @isTest
 public class YetAnotherTestClass {
 	@isTest
-	testMethod static void someTestMethodWithAsserts() {
+	static void someTestMethodWithAsserts() {
```

Comment:
This started showing as a new rule violation and appears to be related to [this fix](https://github.com/pmd/pmd/issues/3183) in PMD. I'm fixing this violation so that the behavior that the tests expect is not modified.